src/main.tsx

```
@@ -1,14 +1,37 @@
+	return (
+		<SortableTree
+			items={treeStructure.items}
+			onItemsChange={setTreeStructure}
+			isRemovable
+			allowNestedItemAddition
+			showDropIndicator
+		></SortableTree>
+	);
+};
+
+createRoot(document.getElementById("root")!).render(
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)